admin/src/hooks/filteringSorting.jsx

```diff
@@ -3,16 +3,16 @@ import { get, set } from 'idb-keyval';
 
 export function useFilter( { slug } ) {
 	const [ currentFilters, setUrl ] = useState( {} );
-	const [ tableQuery, setTableQuery ] = useState( {} );
+	// const [ tableQuery, setTableQuery ] = useState( {} );
 	let filters = '';
 
-	useEffect( () => {
-		get( slug ).then( ( query ) => {
-			const q = query?.currentFilters;
-			setUrl( Object.keys( q ).length ? q : {} );
-			setTableQuery( query );
-		} );
-	}, [ slug ] );
+	// useEffect( () => {
+	// 	get( slug ).then( ( query ) => {
+	// 		const q = query?.currentFilters;
```

Comment:
do we need these comments in the future?

Reply:
will change to other behaviour later